routing/http/server/server.go

```diff
@@ -47,18 +47,26 @@ const (
 var logger = logging.Logger("routing/http/server")
 
 const (
-	providePath       = "/routing/v1/providers/"
-	findProvidersPath = "/routing/v1/providers/{cid}"
-	findPeersPath     = "/routing/v1/peers/{peer-id}"
-	GetIPNSPath       = "/routing/v1/ipns/{cid}"
+	providePath         = "/routing/v1/providers/"
+	findProvidersPath   = "/routing/v1/providers/{cid}"
+	findPeersPath       = "/routing/v1/peers/{peer-id}"
```

Comment:
```suggestion
	findPeersPath       = "/routing/v1/peers/{cid}"
```

see https://github.com/ipfs/specs/pull/476#discussion_r2391931969

Reply:
Switched to `{key}` in https://github.com/ipfs/boxo/pull/1021/commits/713b7c02471cb20f9287f143cd2bbc34d1771a03 (which should correctly work with both peerids and cids)